package.json

```diff
@@ -25,7 +34,6 @@
     "start": "cross-env NODE_ENV=development nodemon node-app.js",
     "start:prod": "node node-app.js"
   },
-  "private": true,
```

Comment:
@mariannemyers it looks like you dropped the private flag, and published grove-node on npm, but it was marked as private on purpose. grove-node is part of the grove templates, not a package.. Let's discuss in the next meeting what we do with this..